specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/apimapis.json

```diff
@@ -1102,6 +1102,224 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/apis/{apiId}/schemas": {
+      "get": {
+        "tags": [
+          "ApiSchema"
+        ],
+        "operationId": "ApiSchema_ListByApi",
```

Comment:
Since this operation returns pageable results I believe you need to annotate it with the x-ms-pageable extension.  See [here](https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md) for more information.

Reply:
nice catch. #fixed